sdk/include/ndk/halfuncs.h

```diff
@@ -32,25 +32,168 @@ Header Name:
+#define HalAllocateMessageTargetOverride                    HALPRIVATEDISPATCH->HalAllocateMessageTargetOverride
+#define HalFreeMessageTargetOverride                        HALPRIVATEDISPATCH->HalFreeMessageTargetOverride
+#define HalDpReplaceBegin                                   HALPRIVATEDISPATCH->HalDpReplaceBegin
+#define HalDpReplaceTarget                                  HALPRIVATEDISPATCH->HalDpReplaceTarget
+#define HalDpReplaceControl                                 HALPRIVATEDISPATCH->HalDpReplaceControl
+#define HalDpReplaceEnd                                     HALPRIVATEDISPATCH->HalDpReplaceEnd
+#define HalPrepareForBugcheck                               HALPRIVATEDISPATCH->HalPrepareForBugcheck
+#endif
+#if (NTDDI_VERSION >= NTDDI_WIN7)
+#define HalQueryWakeTime                                    HALPRIVATEDISPATCH->HalQueryWakeTime
+#define HalReportIdleStateUsage                             HALPRIVATEDISPATCH->HalReportIdleStateUsage
+#define HalTscSynchronization                               HALPRIVATEDISPATCH->HalTscSynchronization
+#define HalWheaInitProcessorGenericSection                  HALPRIVATEDISPATCH->HalWheaInitProcessorGenericSection
+#define HalStopLegacyUsbInterrupts                          HALPRIVATEDISPATCH->HalStopLegacyUsbInterrupts
+#endif
+#if (NTDDI_VERSION >= NTDDI_VISTASP2)
+#define HalReadWheaPhysicalMemory                           HALPRIVATEDISPATCH->HalReadWheaPhysicalMemory
+#define HalWriteWheaPhysicalMemory                          HALPRIVATEDISPATCH->HalWriteWheaPhysicalMemory
+#endif
+#if (NTDDI_VERSION >= NTDDI_WIN7)
+#define HalDpMaskLevelTriggeredInterrupts                   HALPRIVATEDISPATCH->HalDpMaskLevelTriggeredInterrupts
+#define HalDpUnmaskLevelTriggeredInterrupts                 HALPRIVATEDISPATCH->HalDpUnmaskLevelTriggeredInterrupts
+#define HalDpGetInterruptReplayState                        HALPRIVATEDISPATCH->HalDpGetInterruptReplayState
+#define HalDpReplayInterrupts                               HALPRIVATEDISPATCH->HalDpReplayInterrupts
+#define HalQueryIoPortAccessSupported                       HALPRIVATEDISPATCH->HalQueryIoPortAccessSupported
+#endif
+#if (NTDDI_VERSION >= NTDDI_WIN8)
+#define KdSetupIntegratedDeviceForDebugging                 HALPRIVATEDISPATCH->KdSetupIntegratedDeviceForDebugging
+#define KdReleaseIntegratedDeviceForDebugging               HALPRIVATEDISPATCH->KdReleaseIntegratedDeviceForDebugging
+#define HalGetEnlightenmentInformation                      HALPRIVATEDISPATCH->HalGetEnlightenmentInformation
+#define HalAllocateEarlyPages                               HALPRIVATEDISPATCH->HalAllocateEarlyPages
+#define HalMapEarlyPages                                    HALPRIVATEDISPATCH->HalMapEarlyPages
+#if (NTDDI_VERSION == NTDDI_WIN8)
+#define HalGetClockOwner                                    HALPRIVATEDISPATCH->HalGetClockOwner
+#define HalGetClockConfiguration                            HALPRIVATEDISPATCH->HalGetClockConfiguration;
```

Comment:
This still needs to be fixed!
```suggestion
#define HalGetClockConfiguration                            HALPRIVATEDISPATCH->HalGetClockConfiguration
```